ros2_batch_job/__main__.py

```diff
@@ -48,7 +48,8 @@
     'pep8',
     'pydocstyle',
     'pyflakes',
-    'flake8',
+    'flake8==2.6.2',
```

Comment:
version 3 was a complete re-write and doesn't have a stable API yet http://flake8.pycqa.org/en/latest/user/python-api.html
There's a shim to the 2.x API but I wasn't able to see how to overload the reporter as is needed [here](https://github.com/ament/ament_lint/pull/63/files#diff-6f9f51ecc5d47adabeb8b56a439b27deR207)
I can watch the releases to see when an API gets added